modules/ROOT/pages/neo4j-admin-neo4j-cli.adoc

```diff
@@ -71,6 +71,7 @@ The server ID can be used to specify a server in Cypher commands.
 | `license`
 | Accept the license agreement. Possible options are `--accept-commercial` for the link:https://neo4j.com/terms/licensing/[commercial] or `--accept-evaluation` for the link:https://neo4j.com/terms/enterprise_us/[evaluation license].
 This command is required to be run before starting the Neo4j Enterprise Edition.
+Starting from Neo4j 2025.04, the command log includes information about how many days ago the license expired.
```

Comment:
Do we call the output of `neo4j-admin server license` the command log? 
I mean it's ok to use this expression 'the cmd log', but in our docs we usually use 'output'. All cases of using _log_ are usually related to logging and monitoring.

Reply:
I think here we are talking about both the command output and the neo4j.log file.

Reply:
I'll rephrase it. It's a good point.